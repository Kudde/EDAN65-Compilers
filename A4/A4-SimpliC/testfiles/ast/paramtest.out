Program
  List
    Fun
      IntType ID="int"
      IdDecl ID="oneparam"
      Param
        List
          ParamDecl
            IntType ID="int"
            IdDecl ID="a"
      Block
        List
    Fun
      IntType ID="int"
      IdDecl ID="twoparams"
      Param
        List
          ParamDecl
            IntType ID="int"
            IdDecl ID="a"
          ParamDecl
            IntType ID="int"
            IdDecl ID="b"
      Block
        List
