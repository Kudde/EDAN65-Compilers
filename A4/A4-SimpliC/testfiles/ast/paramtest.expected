Program
  List
    Fun
      TypeDecl Type="int"
      IdDecl ID="oneparam"
      Param
        List
          ParamDecl
            TypeDecl Type="int"
            IdDecl ID="a"
      Block
        List
    Fun
      TypeDecl Type="int"
      IdDecl ID="twoparams"
      Param
        List
          ParamDecl
            TypeDecl Type="int"
            IdDecl ID="a"
          ParamDecl
            TypeDecl Type="int"
            IdDecl ID="b"
      Block
        List
